pyaedt/emit_core/results/results.py

```diff
@@ -38,27 +38,22 @@ def __init__(self, emit_obj):
 
     @pyaedt_function_handler()
     def _add_revision(self, name=None):
-        """Add a new revision.
+        """Add a new revision or get the current revision if it already exists.
 
         Parameters
         ----------
         name : str, optional
-            Name for the new revision. If None, it will
-            be named the current design revision.
+            Name for the new revision, if created. If name is ``None``, it will
+            be named after the current design revision.
+
+        Raises
+        ------
+        RuntimeException if the name given is an ex
```

Comment:
Incomplete sentence? This should be a complete sentence ending with a period (full stop).